Add user_posts permission

i-am-ains-ly opened the pull request:
Package was throwing an error when requesting the user_posts permission, which is a completely valid permission as listed in the documentation: https://developers.facebook.com/docs/facebook-login/permissions/v2.0#reference-user_posts

logical-and commented:
Can you please open pull request at [updated library](https://github.com/logical-and/php-oauth), so I merge your changes.

Lusitanian commented:
Can't add in that error reporting code, but without that, could merge.

elliotchance commented on src/OAuth/OAuth2/Service/Facebook.php:
```diff
@@ -10,6 +10,7 @@
 use OAuth\Common\Http\Client\ClientInterface;
 use OAuth\Common\Storage\TokenStorageInterface;
 use OAuth\Common\Http\Uri\UriInterface;
+use RC\Modules\DebugModule\DebugComponent;
```
Remove this.

elliotchance commented:
Please pull in the latest master to fix the build errors.

i-am-ains-ly commented:
Have now removed that redundant bit of error reporting code and reverted other small changes to sync back with the master branch. Also added the new publish_pages permission for Facebook. Can I get a merge ?

elliotchance commented:
There is no diff now. It's just one blank line... Is it possible your changes went in with a previously merged PR?

i-am-ains-ly commented:
Ahh, correct, those changes have been merged by other contributors!

Thanks anyways dude

> On 5 Oct 2015, at 23:46, Elliot Chance <notifications@github.com> wrote:
> 
> There is no diff now. It's just one blank line... Is it possible your changes went in with a previously merged PR?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Lusitanian/PHPoAuthLib/pull/377#issuecomment-145690473>.
>

elliotchance commented:
No worries :smile:
